client/src/components/reviews/AddReview.jsx

```diff
@@ -95,16 +76,14 @@ const AddReview = ({ productId, showModal, openModal, fetchData }) => {
             <option value="1">1 Star - Poor</option>
           </select>
           <p>Do you recommend this product?*</p>
-          <form onChange={handleChange}>
```

Comment:
I initially thought I had to wrap radio buttons in a `<form>` tag so that it knows to deselect one button when the other is selected. Then I realized. that as long as you give both radio buttons the same `name` value, it will know that the two buttons are related.